src/io/flutter/editor/WidgetIndentsHighlightingPassFactory.java

```diff
@@ -26,6 +26,7 @@
 import com.intellij.openapi.util.io.FileUtil;
 import com.intellij.openapi.vfs.VirtualFile;
 import com.intellij.psi.PsiFile;
+import com.jetbrains.lang.dart.analyzer.DartClosingLabelManager;
```

Comment:
based on this PR building through, it looks like this class exists in all Dart plugin versions we support